apps/meteor/server/models/raw/Users.js

```diff
@@ -713,8 +713,10 @@ export class UsersRaw extends BaseRaw {
 						$nin: exceptions,
 					},
 				},
+				{
+					...conditions,
+				},
```

Comment:
Kind of both. With this change we can send custom `$and` or `$or` operations in the query. Without it, neither would work because the query already had both of those keys. 

One of those operations is being used in a query, but I just realized that Alek moved that query into a different PR but left this fix on this one.